src/index.html

```diff
@@ -1,13 +1,74 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/index.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__list">
+        <img
+          class="card__photo"
+          src="images/imac.jpeg"
+          alt="imac"
+        >
+
+        <h2 class="card__product">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <div class="card__text">
+          Product code: 195434
+        </div>
+
+        <div class="card__review">
+          <div class="card__stars">
+            <div class="card__star"></div>
+
+            <div class="card__star"></div>
+
+            <div class="card__star"></div>
+
+            <div class="card__star"></div>
+
+            <div class="card__star"></div>
+          </div>
+
+          <div class="card__rating">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-price">
+            <div class="price">
+              Price:
+            </div>
```

Comment:
p

Reply:
I know div and p tag are block element but when i change it div to p my code doesn't work. Didn't get why